server/src/main/java/org/elasticsearch/transport/TcpTransport.java

```diff
@@ -1018,6 +1018,11 @@ public final TransportStats getStats() {
      */
     public static Set<ProfileSettings> getProfileSettings(Settings settings) {
         HashSet<ProfileSettings> profiles = new HashSet<>();
+        // Process remote cluster port first so that errors are consistently reported if there
+        // is direct usage of the _remote_cluster profile
+        if (REMOTE_CLUSTER_PORT_ENABLED.get(settings)) {
+            profiles.add(RemoteClusterPortSettings.buildRemoteAccessProfileSettings(settings));
+        }
```

Comment:
This is a minor improvement so that we don't get error messages like 
> profile [_remote_cluster] has no port configured

which is confusing because it seems that direct usage of the `_remote_cluster` profile is allowed.